openlibrary/plugins/upstream/models.py

```diff
@@ -141,11 +141,7 @@ def is_daisy_encrypted(self):
         if not meta_fields:
             return
         v = meta_fields['collection']
-        return 'printdisabled' in v or 'lendinglibrary' in v
-
-    #      def is_lending_library(self):
-    #         collections = self.get_ia_collections()
-    #         return 'lendinglibrary' in collections
+        return 'printdisabled' in v
```

Comment:
This looks like it should be

            return 'printdisabled' in v or 'inlibrary' in v

But this will not have been working correctly for some time.

Is this method still current?

It is used in this template here: https://github.com/internetarchive/openlibrary/blob/8b0f5ecdd1cd7e7088c19ac8e3e1d60eca1a95eb/openlibrary/templates/books/daisy.html

I don't know how to access this from the current site.
This FAQ page is still linked: https://openlibrary.org/help/faq/accessing 
and AFAIK DAISY is still relevant and usable?


Two possible courses of action:
1. Fix this code if it is still relevant
2. Delete this method and the corresponding template if it is not current.

Reply:
Raised #9439 for clearing up what is required for DAISY.